docs/src/markdown/extensions/magiclink.md

```diff
@@ -479,6 +480,51 @@ and will copy them for your custom repository. If this is not sufficient, you ca
 `shortener_user_exclude` for your custom repository provider using your specified `name`. If you manually set excludes
 in this manner, no excludes from the same `type` will be copied over.
 
+## Simple User-Defined References
+
+You can integrate with additional services like ticketing systems or shortlink providers (sometimes called `go/` links) by defining simple patterns that will be autolinked.
+
+The `simplerefs` configuration supports matching a static prefix followed by an alphanumeric identifier. We look for patterns of form `{ref_prefix}{identifier}` with the following restrictions:
+
+- `ref_prefix` is a configured literal value consisting of a letter followed by alphanumeric characters or punctuation from the set `_/-`; it will usually end with punctuation like `TICKET-`
```

Comment:
I should also note, is there some restriction of `/`? Number of times I can add `/`? Can I just have a super long `really/really/really/really/really/really/long` reference?

Reply:
I don't see why not as long as it doesn't overlap with other prefixes.

Reply:
Is this something that `go/link` provides? Or is it normally limited to simple IDs. I'm not really big on the idea of allowing infinite `/`.

Reply:
> Is this something that go/ link provides?

For a `go/` link service, generally you can provide links like `go/foo/bar` but in that case it would follow the configured destination for `foo` and append `/bar` to it as a path underneath that destination. So the prefix part is still `go/` and the `foo/bar` would be the identifier part. I consider that a somewhat advanced feature, though, and it's acceptable to not support it (which the proposal here does not).

I just checked Github autolink references, which allowed me to configure a prefix `hi/hi/hi/hi/`.

Reply:
> I just checked Github autolink references, which allowed me to configure a prefix `hi/hi/hi/hi/`.

That's fine, but I don't think we have to completely mirror what GitHub does, but knowing these details is useful.